econml/inference/_inference.py

```diff
@@ -928,9 +918,9 @@ class EmpiricalInferenceResults(InferenceResults):
         The transform function to get the corresponding feature names from featurizer
     """
 
-    def __init__(self, d_t, d_y, pred, pred_dist, inf_type, fname_transformer=lambda nm: nm,
+    def __init__(self, d_t, d_y, pred, pred_dist, inf_type, fname_transformer=None,
                  feature_names=None, output_names=None, treatment_names=None):
-        self.pred_dist = pred_dist
+        self.pred_dist = copy.deepcopy(pred_dist)
```

Comment:
Same comment about copying

```suggestion
        self.pred_dist = np.copy(pred_dist)
```

Reply:
same as above